ts-apps/decision-logic-visualizer/src/lib/data/viz-expr-to-lir.ts

```diff
@@ -114,10 +120,10 @@ function transform(
       const boolvar = new BoolVarLirNode(nodeInfo, originalVar)
       const graph = vertex(boolvar.getId())
       const newEnv = new Map(env).set(originalVar.id, graph)
-      return { graph, vizExprToLir: newEnv }
+      return { graph, vizExprToLirGraph: newEnv }
```

Comment:
This following basically fixes a bug from the previous PR in the threading of the vizExprToLirGraph